test/e2e/multik8s/testutil/util.go

```diff
@@ -258,61 +285,39 @@ func GetPVList(clusterNo int) (*corev1.PersistentVolumeList, error) {
 	return GetObjectList[corev1.PersistentVolumeList](clusterNo, "pv", "")
 }
 
-func ChangeClusterRole(clusterNo int, newRole string) error {
-	deploy, err := GetDeploy(clusterNo, CephClusterNamespace, MantleControllerDeployName)
-	if err != nil {
-		return fmt.Errorf("failed to get mantle-controller deploy: %w", err)
-	}
-
-	roleIndex := slices.IndexFunc(
-		deploy.Spec.Template.Spec.Containers[0].Args,
-		func(arg string) bool { return strings.HasPrefix(arg, "--role=") },
-	)
-	if roleIndex == -1 {
-		return errors.New("failed to find --role= argument")
-	}
-
-	_, _, err = Kubectl(
-		clusterNo, nil, "patch", "deploy", "-n", CephClusterNamespace, MantleControllerDeployName, "--type=json",
-		fmt.Sprintf(
-			`-p=[{"op": "replace", "path": "/spec/template/spec/containers/0/args/%d", "value":"--role=%s"}]`,
-			roleIndex,
-			newRole,
-		),
-	)
-	if err != nil {
-		return fmt.Errorf("failed to patch mantle-controller deploy: %w", err)
-	}
+func GetEventList(clusterNo int, namespace string) (*corev1.EventList, error) {
+	return GetObjectList[corev1.EventList](clusterNo, "event", namespace)
+}
 
-	// Wait for the new controller to start
-	numRetries := 10
-	for i := 0; i < numRetries; i++ {
-		stdout, _, err := Kubectl(clusterNo, nil, "get", "pod", "-n", CephClusterNamespace, "-o", "json")
+func ChangeClusterRole(clusterNo int, newRole string) error {
+	switch clusterNo {
+	case PrimaryK8sCluster:
+		err := Make("minikube-profile-primary")
```

Comment:
Extracting profileTarget as a variable eliminate duplicated Make() calls.

```suggestion
	var profileTarget string
	switch clusterNo {
	case PrimaryK8sCluster:
		profileTarget = "minikube-profile-primary"
```